Sources/App/Extensions/Router+Extensions.swift

```diff
@@ -10,23 +10,19 @@ import Vapor
 extension Request {
     var acceptLanguage: String {
         get {
-            guard let accpetLanguage = self.http
+            let accpetLanguage = String(self.http
                 .headers
                 .firstValue(name: .acceptLanguage)?
                 .split(separator: ",")
-                .first else {
+                .first?
+                .split(separator: "-")
+                .first ?? "en")
+            let supportedLanguages = ["ar", "de", "en", "fa", "fr", "id", "ja", "ko", "pt", "ro", "ru", "tr", "zh"]
```

Comment:
I guess we will have to update this every time someone submits a new localisation file, right?

Reply:
I think it would be good to add README.md file to the Localizations directory explaining that.

Reply:
Added, please review @adie @gnechita-toptal